doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -195,7 +197,7 @@ Reference to "Architecture" in this chapter refers to the NFVI Hardware (e.g. ph
 | `req.inf.stg.04` | Storage | The Architecture **may** support Software Defined Storage (SDS) that seamlessly supports shared block storage, object storage and flat files. |
 | `req.inf.stg.05` | Storage | The Architecture **should** provide high-performance and horizontally scalable storage. |
 | `req.inf.stg.06` | Storage | The Architecture **must** support ephemeral storage (non-persistent) storage for Pods. |
-| `req.inf.stg.07` | Storage | The Architecture **must** support persistent storage for Pods. |
+| `req.inf.stg.07` | Storage | The Architecture **must** support persistent storage for Pods (*optional for deployment*). |
```

Comment:
I propose an **additional** requirement to make the intention of the previous one unambiguously:

req.infr.stg.08 The architecture **must** allow the operator the choice during deployment to not deploy the support of persistent storage.

Reply:
Ok, yes this kind of wording works for me as we're asking the architecture to support that optionality (as opposed to the target of the requirement being the implementation or deployment).

Reply:
@peterwoerndle , @fkautz what do you think?

Reply:
should: Requirements that are marked as should are expected to be fulfilled by the reference architecture but it is up to each service provider to accept an implementation targeting this reference architecture that is **not reflecting** on any of those requirements. 

The above definition could be better worded but doesn't "should" capture what is being proposed?

Reply:
I don't think so. We don't want the reference or vendor Implementations to not reflect the requirement, which the wording you've provided would allow. I.e. a vendor implementation could not deliver it and still conform to the specification. Our end goal is that the implementation must reflect the requirement but that the capability is optional when deploying.

Reply:
@tomkivlin then the simpler method would be to have a "must" requirement for implementation and another for "may" requirement for deployment

Reply:
Which is fine, in those documents. I don't think its appropriate to have a requirement in the RA that has the implementation or deployment as the target of the requirement as we wouldn't be able to test conformance of it. I like Herbert's version as it's about ensuring the RA provides the ability/flexibility for it to be optional during a deployment. That can be tested by checking for dependencies within the specification.

Reply:
Herbert's version makes it really unambiguous.  Let's do it that way.  And follow this path with other requirements that are "optional for deployment".

Reply:
I have updated in [c8e8857](https://github.com/cntt-n/CNTT/pull/1130/commits/c8e8857ab348d40520c2cfdf7b6d9284e12ab422) - if you're happy @hdamker could you resolve both conversations and approve the PR please?

Reply:
@tomkivlin Thanks, I'm happy ;-)